src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -337,7 +337,7 @@ function RemoteFunctions(experimental, remoteWSPort) {
                 _trigger(element, "highlight", 1);
             }
             
-            if (!window.event && !isInViewport(element)) {
+            if ((!window.event || window.event instanceof MessageEvent) && !isInViewport(element)) {
```

Comment:
check for `window.event instanceof MessageEvent` is added because in latest version of chrome, when it receives a message through websocket, then it considers it as window event and thus window.event is true, so we are bypassing such scenario by checking if the event is of type Message, then we can safely ignore this.